test/multiverse/suites/rails/rails_logger_test.rb

```diff
@@ -0,0 +1,43 @@
+# This file is distributed under New Relic's license terms.
```

Comment:
Hopefully, this test will give us a heads-up if something changes with the Rails logger that causes the duplication issue to resurface.